en/StopTimeUpdate.md

```diff
@@ -0,0 +1,12 @@
+### StopTimeUpdate
+
+| Field Name | Recommendation |
+| --- | --- |
+| stop_sequence | If `stop_sequence` and `stop_id` are both provided, any combination of `stop_sequence` and `stop_id` must have a match in GTFS data. |
+| stop_id | All `stop_ids`  must exist in the GTFS data in `stops.txt` and must have `location_type=0` in `stops.txt`. |
+|  | If `stop_sequence` and `stop_id` are both provided, any combination of `stop_sequence` and `stop_id` must have a match in GTFS data. |
+| arrival | Arrival times between sequential stops should always increase - they should never be the same or decrease. Arrival `time` (specified in [StopTimeEvent](#StopTimeEvent)) should be before departure `time`. |

```

Comment:
"Should always" and "should never" are confounding. It's not clear if these are suggestions or obligations.

Reply:
Fixed in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/1/commits/5aa9bd712583e02fd1b14bfff19a72fbb295ce55. This should belong in the spec as a "MUST" - I added to #5. But here I'm changing to SHOULD so we can leave it in best practices.